schemas/string_interpolation/definitions.json

```diff
@@ -0,0 +1,143 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "string_with_placeholders": {
+    "oneOf": [
+      {
+        "description": "String with no placeholders.",
+        "type": "string"
+      },
+      {
+        "description": "An object that represents a string with placeholders.",
+        "type": "object",
+        "properties": {
+          "text": {
+            "type": "string"
+          },
+          "placeholders": {
+            "$ref": "#/placeholders"
+          },
+          "additionalProperties": false,
+          "required": [
+            "text",
+            "placeholders"
```

Comment:
These don't seem to be enforced for me. If I define:
```
"title": {
	"text": "hello"
}
This passes validator for me.

Reply:
Good spot! `additionalProperties` and `required` were at the wrong level in the schema. Fixed.